src/main/java/streamingapi/client/StreamingApiClient.java

```diff
@@ -97,33 +108,43 @@ public Subscription createSubscription(Subscription subscription, String streami
 
 	/**
 	 * Consume events of based on a subscription.
+	 * When the connection to streaming api is lost, the connection is reinitialized if the monitor approves.
```

Comment:
I would rephrase the Info to: 
The connection to streaming api can be lost when the streaming api events store closes the stream because the stream timeout is reached or due to unpredictable network events.